### PR TITLE
Kartograf integration

### PR DESCRIPTION
**Pull request by RiesBen**
future integration of atom_mapper, etc.

### CODE REVIEW

File: openfe/setup/__init__.py [hunk 1]
```diff
@@ -5,7 +5,8 @@
 from .atom_mapping import (LigandAtomMapping,
                            LigandAtomMapper,
                            LomapAtomMapper, lomap_scorers,
-                           PersesAtomMapper, perses_scorers)
+                           PersesAtomMapper, perses_scorers,
+                           KartografAtomMapper)
```
Comment (mikemhenry) [inline openfe/setup/__init__.py:9]:
```suggestion
                           KartografAtomMapper,)
```

File: docs/environment.yaml [hunk 2]
```diff
@@ -25,6 +25,7 @@ dependencies:
   - sphinx-toolbox
   - sphinx<7
   - git+https://github.com/OpenFreeEnergy/gufe@main
+  - git+https://github.com/OpenFreeEnergy/kartograf@main
```
Comment (richardjgowers) [inline docs/environment.yaml:28]: use conda package not a pip install

Comment (RiesBen) [reply docs/environment.yaml]: check

**Review by richardjgowers — APPROVED** [review]

**Comment by richardjgowers** [comment]
for some reason installing kartograf downgraded gufe to 0.7.3...
```
In [2]: import gufe

In [3]: gufe.__version__
Out[3]: '0.7.3'
```

**Comment by richardjgowers** [comment]
I think this was an openff-models pin in gufe, have updated that pin in the feedstock

**Pull request merged**